src/themes/circuit.js

```diff
@@ -221,16 +221,19 @@ const breakpoints = {
 
 export const mq = createMediaQueries(breakpoints);
 
-export const transitions = {
-  default: `200ms ease-in-out`
-};
+export { default as transitions } from '../styles/transitions';
```

Comment:
Design still hasn't come up with transitions and we don't know what the appropriate structure for these will be. Please don't export these here and encapsulate them for the components you created. That was the reason why there only was a single very silly transition before. We just don't know what is going to happen with them and exposing a ton of different transitions now is just going to cause more headaches and debt if people start using them.